editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -2259,8 +2259,6 @@ void CanvasItemEditor::_gui_input_viewport(const Ref<InputEvent> &p_event) {
 		case DRAG_MOVE:
 			c = CURSOR_MOVE;
 			break;
-		case DRAG_PAN:
-			c = CURSOR_DRAG;
```

Comment:
Needs rebase as I added a `break:` here, so there's a merge conflict.

Reply:
Please also squash the commits together.

Reply:
This merge messed up everything again. I just give up .-.
I'll make a new PR soon.